.travis.yml

```diff
@@ -3,6 +3,34 @@ go_import_path: github.com/ethereum/go-ethereum
 sudo: false
 matrix:
   include:
+        # This builder only tests code linters on latest version of Go
+    - stage: lint
+      os: linux
+      dist: xenial
+      go: 1.13.x
+      env:
+        - lint
+      git:
+        submodules: false # avoid cloning ethereum/tests
+      script:
+        - go run build/ci.go lint
+
+    - stage: build
+      os: linux
+      dist: xenial
+      go: 1.11.x
+      script:
+        - go run build/ci.go install
+        - go run build/ci.go test -coverage $TEST_PACKAGES
```

Comment:
Better use `travis_wait` here same as below (build defined at `L35`). Otherwise it can time-out.

Or you can just remove those two builds as it's identical except the golang compiler version.